hazelcast/src/test/java/com/hazelcast/test/annotation/SlowTest.java

```diff
@@ -20,6 +20,8 @@
  * Annotates tests which are too slow for the PR builder.
  * <p>
  * Will be executed in nightly builds and for code coverage measurements.
+ * <p>
+ * The opposite of a {@link QuickTest}
```

Comment:
"nightly builds and for code coverage" != !"PR builder and for code coverage"

Reply:
> "nightly builds and for code coverage" != !"PR builder and for code coverage"

Do you have a suggested alternative?
This isn't something I've changed?

Reply:
I referred to "opposite". They are not strictly opposite, third alternative being `NightlyTest`, so "The opposite of a `{@link QuickTest}`" might be misleading.

Reply:
> I referred to "opposite". They are not strictly opposite, third alternative being `NightlyTest`, so "The opposite of a `{@link QuickTest}`" might be misleading.

Good spot, completely missed `NightlyTest`

180e8d852810ad387f5e5ff7ac45f94bb8106e31